src/instance_tracker.ts

```diff
@@ -222,6 +242,19 @@ export class InstanceTracker {
             JSON.stringify(state),
         );
 
+        if (reconfigureValue !== null) {
+            returnReconfigureValue = true;
+            // reconfigure was set, so check report for last reconfigured
+            if (state.lastReconfigured) {
+                const dLast = new Date(state.lastReconfigured);
+                const dValue = new Date(reconfigureValue);
+                if (dLast >= dValue) {
```

Comment:
I see your point, and I think it's reasonable.  However, the behavior I'm looking for is "reconfigured once since this time", even if multiple reconfigure requests come in rapid succession (as is likely to happen during a release).
The way I hope to mitigate this is to change the reconfiguration boolean to pass the new reconfigure date from the server, and store/reply with that from the sidecar.  That way we allow the timestamp to be server-sourced but still maintain the "once since this time" behavior.